www/ui_components/server-card.html

```diff
@@ -57,6 +57,7 @@
       }
 
       .card-content {
+        padding: 20% 0 20% 0;
```

Comment:
This makes the card too big for smaller displays - I believe the smallest we target is iPhone 4 (320x480).

<img width="375" alt="screen shot 2018-03-23 at 6 13 24 pm" src="https://user-images.githubusercontent.com/2132122/37855517-07bba72a-2ec6-11e8-873d-6e6448fca426.png">

I played around with some values and suggest we use the following rules:
```css 
     .card-content {
        text-align: center;
        padding: 10% 0;
      }

      @media (min-height: 600px) {
        .card-content {
          padding: 20% 0;
        }
      }
```
What do you think?

Reply:
added